broker/l2tp_broker.cfg.example

```diff
@@ -5,13 +5,15 @@ address=127.0.0.1
 port=53,123,8942
 ; Interface with that IP address
 interface=lo
-; Maximum number of tunnels that will be allowed by the broker
-max_tunnels=1024
+; Maximum number of tunnels that will be allowed by the broker.
+; On a cheap VPS, more than 256 tunnels usually do not make sense.
+; Physical machines can take more.
+max_tunnels=256
 ; Tunnel id base
 tunnel_id_base=100
 ; Reject connections if there are less than N seconds since the last connection.
 ; Can be less than a second (e.g., 0.1).
-connection_rate_limit=10
+connection_rate_limit=0.2
```

Comment:
Hm, maybe we should have rate limiting per source IP, not global. Otherwise rate limiting is really hard.

Reply:
@mitar  You mean something like https://github.com/wlanslovenija/tunneldigger/pull/138? :D

Reply:
:-) So let's update then defaults then. :-)